src/Components/GEOSldas_GridComp/GEOSlandpert_GridComp/LDAS_PertRoutines.F90

```diff
@@ -1150,11 +1154,11 @@ subroutine get_force_pert_inputs( pert_grid_l,                         &
                         )
                    if (nc4_stat /= nf90_noerr) call handle_nc4_stat(nc4_stat)
                    ! get _FillValue for nc4_varname
-                   nc4_stat = nf90_get_att(nc4_grpid, nc4_varid, '_FillValue', nc4_fillval)
-                   if (nc4_stat /= nf90_noerr) call handle_nc4_stat(nc4_stat)
+                   !nc4_stat = nf90_get_att(nc4_grpid, nc4_varid, '_FillValue', nc4_fillval)
+                   !if (nc4_stat /= nf90_noerr) call handle_nc4_stat(nc4_stat)
                    ! replace _FillValue by zero
-                   where (abs(std_force_pert(ivar,:,:)-nc4_fillval)<nodata_tolfrac_generic) &
-                        std_force_pert(ivar,:,:) = 0.
+                   !where (abs(std_force_pert(ivar,:,:)-nc4_fillval)<nodata_tolfrac_generic) &
+                   !    std_force_pert(ivar,:,:) = 0.
```

Comment:
@saraqzhang, 
Why comment out these lines?  Without them, the input file will have to have physically sensible values of, say, std_progn_pert for all grid cells, including ocean.   I suppose we could construct input files that have 0 over ocean.  If we do that, then these lines don't do anything.  But if the std_progn_pert inputs contain 1.e15 or -9999, chances are that subsequent calculations using very large or negative numbers might crash.  The lines in question here should prevent this from happening.

Reply:
@gmao-rreichle ,  the current file is created with  the default  sdv value as the base  ( the numbers in ensprop.nml ) everywhere,  then adding modification to certain region. I have not included land mask information in the python code I  made, therefore no  attribute fill value set in the file.  We certainly can add  fillval to attributes when creating the file. and keep those fillval lines the code here.

Reply:
@gmao-rreichle ,  the fill value attributes are added to the sdv nc4 file.  The lines reading  fill values are reinstalled and tested with the new nc4 file input.   the comment "!' on those lines can be removed.  I pushed the changes to this branch, but don't know how this PR  be updated.

Reply:
@saraqzhang, thanks, no need for further updates.  By pushing the branch to github, the PR automatically updates.  Since the only difference (other than added comments) is the location of the nc4 close statement that doesn't get used in any of the tests, the PR should be trivially 0-diff.  I'll work with @biljanaorescanin on when/how we can best merge it.  It may be best to bundle it with another branch/PR that will be tested for 0-diff, just to be safe.